### DISCUSSION TITLE
Milestone developments know which issues will be resolved

### DISCUSSION THREAD

**Discussion by Angelk90** [zed-industries/zed#19595] [Feature Requests]
Hi, It would be useful to be able to use projects to know the development of the current `Milestone` and future ones.

Which issues will be solved in the current `Milestone` and in the future ones, that is, it allows to be more organized in the development of the project.

For example, taking a repository that uses correctly, [CE](https://github.com/orgs/CodeEditApp/projects/3/views/23) you can see in the current milestone which are `todo`, `in progress` and `complete`.

<img width="1428" alt="Screenshot 2024-10-22 alle 19 01 12" src="https://github.com/user-attachments/assets/525b98fc-2410-4ef4-867e-19b6ec3f4a5e">

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
